page-objects/createCourse.js

```diff
@@ -37,9 +38,54 @@ module.exports = {
 		let nameSelector = await driver.$('[data-testid="coursename"]');
 		await nameSelector.setValue(coursename)
 	},
-	setColour: async function() {
-		let inputColor = await driver.$(courseData.elem.colorCourse);
-		await inputColor.click();
+	setColour: async function(name) {
```

Comment:
There's a lot of repeating code here. A cleaner approach could be:
```
const colors = ["grey","metallicGold","blue","green","darkGrey","goldenPoppy","martini","violetRed","corn"]
const child = colors.indexOf(name) + 1
if (child <= 0) {
console.warn(`you did not insert a valid color. Must be ${colors},\n you inserted ${name}`)
const color = await driver.$(`.color-picker__item:nth-child(${child})`);
await color.click();
```

Reply:
Thank you, Tor, a nice point!